test/test_ownership_id.py

```diff
@@ -0,0 +1,77 @@
+import json
+from unittest.mock import patch
+from test.helper import web
+from test import with_context
+from test import db, Fixtures
+from test.factories import ProjectFactory, UserFactory
+from pybossa.core import user_repo
+from pybossa.repositories import ProjectRepository
+
+
+class TestOwnershipId(web.Helper):
```

Comment:
If the project info API can be used in place of the new ownership endpoint, both the ownership endpoint and unit tests could be removed.